tests/datalab/issue_manager/test_knn_graph.py

```diff
@@ -0,0 +1,66 @@
+import math
+from hypothesis import HealthCheck, given, settings, strategies as st
+import pytest
+
+from .conftest import knn_graph_strategy
+
+
+class TestKNNGraph:
```

Comment:
question: this test is not for testing cleanlab functionality right? it's just to check that the `knn_graph_strategy` gives KNNs with the right properties?